### PR TITLE
Check formplayer serverup

### PR DESCRIPTION
**Pull request by benrudolph**
@wpride @dannyroberts this adds a serverup check for formplayer (and fixes a bug where it didn't work locally)

### CODE REVIEW

File: corehq/apps/hqadmin/service_checks.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def check_couch():
 
 
 def check_formplayer():
-    formplayer_url = '/formplayer' or settings.FORMPLAYER_URL
```
Comment (dannyroberts) [inline corehq/apps/hqadmin/service_checks.py]: Lol

**Comment by dannyroberts** [comment]
👍 lgtm

File: corehq/apps/hqadmin/service_checks.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/hqadmin/service_checks.py]: 😳 was using it for testing and it must've gotten committed

**Comment by benrudolph** [comment]
✔️

**Pull request merged**